ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/deprovision/cucushift-installer-rehearse-aws-ipi-deprovision-chain.yaml

```diff
@@ -5,5 +5,6 @@ chain:
   - chain: gather-core-dump
   - ref: gather-aws-console
   - chain: ipi-deprovision
+  - ref: aws-deprovision-users-and-policies
```

Comment:
Sounds like we can add `aws-deprovision-users-and-policies` into `ipi-deprovision` chain.

Reply:
That will affect other platforms, as `aws-deprovision-users-and-policies` is only applicable to AWS

Reply:
ah, good point.